docs/general-settings.md

```diff
@@ -20,5 +20,10 @@ settings: {
 * **header.logo** defines the top left navigation logo. It has a fixed height of 28px.
 * **header.title** defines the top left navigation title.
 * **header.favicon** defines the favicon. It rquires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
-* **responsiveNavigation** allows adding a button on the left side of the top navigation. Upon click, the button shows or hides the left navigation. The possible values are `simple` and `simpleMobileOnly`. `simple` displays the button regardless of the browser window´s size, while `simpleMobileOnly` shows the button when the width is lower than `600px`. If you don't specify any value for  **responsiveNavigation**, the button will not be displayed. The same applies when you enable **hideSideNav** for the current active navigation node. 
-* **sideNavFooterText** is a string displayed in a sticky footer inside the Side Navigation. It is a good place to display the version of your application.
+* **responsiveNavigation** allows customizing the navigation display settings. For example, you can define buttons which show or hide the left navigation completely, or collapse it to only show the icons. 
```

Comment:
```suggestion
* **responsiveNavigation** allows customizing the navigation display settings. For example, you can define buttons which show or completely hide the left navigation, or collapse it to only show the icons. 
```
Can we move "completely" so that it refers only to "hide"?